packages/@aws-cdk/aws-backup/README.md

```diff
@@ -82,6 +82,15 @@ const plan = backup.BackupPlan.daily35DayRetention(this, 'Plan', myVault); // Us
 plan.addRule(backup.BackupPlanRule.monthly1Year(otherVault)); // Use `otherVault` for this specific rule
 ```
 
+you can backup VSS(Volume Shadow Copy Service)-enabled Windows applications running on Amazon EC2 instances. If the application has VSS writer registered with Windows VSS, then AWS Backup creates a snapshot that will be consistent for that application.
+
+This can be used by the windowsVss parameter, and the default value of this parameter is false.
```

Comment:
I think we can delete this line

Reply:
Previously, another maintainer asked me to update the additional explanation to readme. Is it okay to delete it?

Reply:
Not the entire explanation. Just the sentence "This can be used by the windowsVss parameter, and the default value of this parameter is false." -- sorry I wasn't more clear.